### ISSUE TITLE
WCAG: close popup modals should be possible with keyboard

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2240]
The button to close a popup is made from a non-interactive div element.

The reading software cannot tell its user that these are buttons because they miss the correct role. Now these buttons are coded as neutral (non-interactive) div elements. In addition, the reading software does not know what the function (name) of these buttons is.

The most robust solution is to use the native HTML elements such as button elements. Another possible solution is to use WAI-ARIA.

![image.png](https://images.zenhubusercontent.com/5ba4ae539f1e9e096ebdcc88/5ce16e78-9839-4265-8baf-b4677d45c6b2)

**Comment by jamalv** [comment]
should be able to close all modals using the "escape" key
